Tasks/AzureResourceManagerTemplateDeploymentV3/operations/DeploymentScopeBase.ts

```diff
@@ -54,6 +54,9 @@ export class DeploymentScopeBase {
                             return this.waitAndPerformAzureDeployment(retryCount);
                         }
                         utils.writeDeploymentErrors(this.taskParameters, error);
+                        if(this.taskParameters.deploymentScope == "Resource Group" || this.taskParameters.deploymentScope == "Subscription") {
```

Comment:
to keep things simple here, you can move these two lines including if-else condition to separate function. this way the code here is just one line. and inside the new function you can merge getAzurePortalDeploymentURL() as well.

Reply:
Done